tools/dhtnode.cpp

```diff
@@ -113,12 +115,7 @@ void cmd_loop(std::shared_ptr<DhtRunner>& dht, dht_params& params)
             std::cout << dht->getRoutingTablesLog(AF_INET6) << std::endl;
             continue;
         } else if (op == "ld") {
-            iss >> idstr;
-            InfoHash filter(idstr);
-            if (filter == InfoHash{})
-                std::cout << dht->getStorageLog() << std::endl;
-            else
-                std::cout << dht->getStorageLog(filter) << std::endl;
+            std::cout << dht->getStorageLog() << std::endl;
```

Comment:
Just to be clear, I think @aberaud is referring to this commit: e3663c995a7704802aafbeb57f0bd48f17e4e7b4